pkg/storkctl/migration_test.go

```diff
@@ -0,0 +1,70 @@
+// +build unittest
+
+package storkctl
+
+import (
+	"testing"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+)
+
+func TestGetMigrationsNoMigration(t *testing.T) {
+	cmdArgs := []string{"migrations"}
+
+	var migrationList storkv1.MigrationList
+	expected := "No resources found.\n"
+	testCommon(t, newGetCommand, cmdArgs, &migrationList, expected, false)
+}
+
+func TestGetMigrationsOneMigration(t *testing.T) {
+	cmdArgs := []string{"migrations"}
+
+	migration := &storkv1.Migration{
+		Spec: storkv1.MigrationSpec{
+			ClusterPair:       "clusterPair1",
+			Namespaces:        []string{"namespace1", "namespace2"},
+			IncludeResources:  true,
+			StartApplications: true,
+		},
+	}
+	migration.Name = "migration1"
+
+	var migrationList storkv1.MigrationList
+	migrationList.Items = append(migrationList.Items, *migration)
+
+	expected := "No resources found.\n"
```

Comment:
This should return the above migration, right?

Reply:
This is where I was getting stuck trying to figure out how to use the `Fake`.  In the debugger, I was hitting this code:

```
=>  51:	func (c *FakeMigrations) List(opts v1.ListOptions) (result *v1alpha1.MigrationList, err error) {
    52:		obj, err := c.Fake.
    53:			Invokes(testing.NewRootListAction(migrationsResource, migrationsKind, opts), &v1alpha1.MigrationList{})
```

in `github.com/libopenstorage/stork/pkg/client/clientset/versioned/typed/stork/v1alpha1/fake/fake_migration.go`

That seems to return an empty migration list.  I couldn't figure out how to prepopulate that.